challenge-13/challenge-13.js

```diff
@@ -1,150 +1,183 @@
+
+	/*
+	Mostre no console todos os estados que estão em `brasil`.
+	*/
+	console.log( '\nAlguns Estados do Brasil:' );
+	console.log( brasil );
+
+	/*
+	Crie um novo array chamado `nordeste`, que tenha os estados do nordeste.
+	*/
+	var nordeste = [ 'Alagoas', 'Bahia', 'Ceará' ];
+
+	/*
+	Mostre no console os estados do nordeste.
+	*/
+	console.log( '\nEstados do Nordeste:' );
+	console.log( nordeste );
+
+	/*
+	Remova de `brasil` os estados do `sudeste`, colocando-os em uma variável
+	chamada `newSudeste`.
+	*/
+	var newSudeste = brasil.splice( 2, 3 );
+
+	/*
+	Adicione os estados do `nordeste` ao array `brasil`. Esses estados devem
+	ficar no mesmo nível que os estados já existentes, não em um array separado.
+	*/
+	brasil = brasil.concat( nordeste );
+
+	/*
+	Mostre no console os estados em `newSudeste`.
+	*/
+	console.log( '\nEstados em newSudeste:' );
+	console.log( newSudeste );
+
+	/*
+	Mostre no console os estados do `brasil`.
+	*/
+	console.log( '\nAlguns estados do Brasil:' );
+	console.log( brasil );
+
+	/*
+	usando forEach, percorra o array `brasil` e gere um novo array chamado
+	`newBrasil`. Esse array deve ter cada item como um objeto, com as
+	propriedades:
+	- `id`: que será o índice do array `brasil`,
+	- `estado`: que será o estado do array `brasil`.
+	*/
+	var newBrasil = [];
+	brasil.forEach( function( element, index, arr ){
+		newBrasil.push( { 'id': index, 'estado': element } );
+	});
+
+	/*
+	Mostre o array `newBrasil` no console
+	*/
+	console.log( '\nnewBrasil:' );
+	console.log( newBrasil );
+
+	/*
+	Percorra o array `brasil` e verifique se os estados tem mais de 7 letras cada,
+	atribuindo o resultado à uma variável. Se tiver, mostre no console a frase:
+	- "Sim, todos os estados tem mais de 7 letras!"
+	Senão, mostre no console:
+	- "Nem todos os estados tem mais de 7 letras!"
+	*/
+	console.log( '\nTodos os estados de `brasil` tem mais de 7 letras?' );
+	var result = brasil.every( function( item ){
+		return item.length > 7;
+	});
+
+	console.log( 
+		result 
+		? 'Sim, todos os estados tem mais de 7 letras!'
+		: 'Nem todos os estados tem mais de 7 letras!'
+	);
+	
+	/*
+	Percorra o array `brasil` e verifique se o Ceará está incluído, atribuindo o
+	resultado à uma variável. Se esse estado existir no array, mostrar a frase no
+	console:
+	- "Ceará está incluído!"
+	Senão, mostrar a frase:
+	- "Ceará não foi incluído :("
+	*/
+	console.log( '\nCeará está incluído em `brasil`?' );
+	var otherResult = brasil.some( function( item ){
+		return item === 'Ceará';
+	});
+	
+	console.log( 
+		otherResult 
+		? 'Ceará está incluído!' 
+		: 'Ceará não foi incluído :('
+	);
+
+	/*
+	Percorra o array `newBrasil` e crie um novo array que some 1 no ID de cada
+	objeto desse array, e adicione a frase abaixo na propriedade `estado`:
+	- "[ESTADO] pertence ao Brasil."
+	Atribua o novo array a uma variável chamada `map`.
+	*/
+	var map = newBrasil.map( function( element, index, arr ) {
+		return { 
+			'id': element.id + 1,
```

Comment:
-1
As propriedades de objetos não precisam de aspas. Somente se você for utilizar caracteres especiais :)